force/sobjects.go

```diff
@@ -112,6 +114,41 @@ func (forceApi *ForceApi) DeleteSObject(id string, in SObject) (err error) {
 	return
 }
 
+func idsFromURIs(uris []string) []string {
```

Comment:
There may be arguments against this, but I think it would be nice to leave a reference to some Salesforce documentation about this in comments for the next person since it's so quirky (that, if there's a 300 status code then multiple URIs are returned such that the end of that is always the ID in question).